fmriprep/workflows/fieldmap/fmap.py

```diff
@@ -96,9 +91,11 @@ def init_fmap_wf(reportlets_dir, omp_nthreads, fmap_bspline, name='fmap_wf'):
 
     else:
         torads = pe.Node(niu.Function(output_names=['out_file', 'cutoff_hz'],
-                                      function=_torads), name='torads')
+                                      function=_torads), name='torads',
+                         run_without_submitting=True)
```

Comment:
We're not using `run_without_submitting` on `Function` nodes at present. See #554.